dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/DefaultDataValueSetService.java

```diff
@@ -1070,6 +1070,12 @@ private ImportContext createDataValueSetImportContext( ImportOptions options, Da
                 || settings.getBoolSetting( SettingKey.DATA_IMPORT_STRICT_ATTRIBUTE_OPTION_COMBOS ) )
             .strictOrgUnits( options.isStrictOrganisationUnits()
                 || settings.getBoolSetting( SettingKey.DATA_IMPORT_STRICT_ORGANISATION_UNITS ) )
+            .strictDataSetApproval( options.isStrictDataSetApproval()
+                || settings.getBoolSetting( SettingKey.DATA_IMPORT_STRICT_DATA_SET_APPROVAL ) )
+            .strictDataSetLocking( options.isStrictDataSetLocking()
+                || settings.getBoolSetting( SettingKey.DATA_IMPORT_STRICT_DATA_SET_LOCKING ) )
+            .strictDataSetLocking( options.isStrictDataSetInputPeriods()
```

Comment:
Copy/Paste/NoEdit error: Should be `.strictDataSetInputPeriods( options...`

Reply:
Nice catch. Fixed.